docs/UserGuide.md

```diff
@@ -47,110 +49,161 @@ Format: `help`
 
 > Help is also shown if you enter an incorrect command e.g. `abcd`
 
-### 2.2. Adding a person: `add`
+### 2.2. Adding a task/event : `add`
+
+Adds a task/event to the address book<br>
+
+Task: 
+
+Format: `add task TASK_NAME [dl/DEADLINE] [des/DESCRIPTION] [t/TAG]...`
+
+> Tasks can have a deadline
+> Tasks can have a description
+> Tasks can have any number of tags (including 0)
+
+
+Examples:
+* `add Do laundry dl/2017-3-1-2359 `
+* `add Study for midterms dl/2017-3-2-2359 des/CS2103 at lt7 `
+* `add Buy milk for baby dl/2017-3-3-2100 des/yaas milk t/family`
+* `add Create user story dl/2017-4-1-1300 t/work t/computing` 
+
+Event: 
+
+Format: `add event EVENT_NAME st/START_TIME et/END_TIME [des/DESCRIPTION] [t/TAG]...`
+
+> * Event can have description
+> * Events can have any number of tags (including 0)
+
+
+Examples:
+* `Add Midterms st/2017-3-1-1300 et/2017-3-1-1500 des/CS2103 t/school`
```

Comment:
on second thoughts, since your examples look the same for both, why not have single command format and decide if it is task or event looking at what is given as input?